Source/WebCore/css/parser/CSSParserContext.h

```diff
@@ -101,8 +101,8 @@ struct CSSParserContext {
     // Settings, those affecting properties.
     CSSPropertySettings propertySettings;
 
-    CSSParserContext(CSSParserMode, const URL& baseURL = URL());
-    WEBCORE_EXPORT CSSParserContext(const Document&, const URL& baseURL = URL(), const String& charset = emptyString());
+    explicit CSSParserContext(CSSParserMode, const URL& baseURL = URL());
```

Comment:
Not sure this change is as valuable as the `Document` one. Seems unlikely this implicit constructor does harm anywhere.

Reply:
I thought it was our policy in WebKit to mark constructors taking a single parameters (or single mandatory one in this case) as explicit. These 2 didn't follow our pattern and one of them caused this subtle bug as a result. This is why it was my personal preference to update both.

Reply:
Well, we mark constructors that aren't just lossless type conversions as explicit. I think it's not perfectly clear that this is one of those cases.